README.md

```diff
@@ -1,24 +1,89 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Rails recipe App
 
-Things you may want to cover:
+>Rails recipe app is a Ruby on Rails application that keeps track of all your foods, recipes, ingredients, and inventories.
 
-* Ruby version
 
-* System dependencies
+## ERD
+![image](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_erd.png)
 
-* Configuration
+### Cloning the project
 
-* Database creation
+git clone https://github.com/ger619/rails-capstone.git <Your-Build-Directory>
+``` 
+- cd rails-capstone
+- rails s
+```
```

Comment:
- [ ] It would be better if we include the live link version or deployed version of our app in the README file :man_technologist: 

![Screenshot from 2022-09-08 18-26-00](https://user-images.githubusercontent.com/46388113/189200133-bf91bef3-84ee-42bb-bf95-3ce1e0930924.png)